Makefile

```diff
@@ -11,8 +11,8 @@ OHMYGLB_IMAGE ?= absaoss/ohmyglb:$(VERSION)
 .PHONY: up-local
 up-local: create-test-ns
 	kubectl apply -f ./deploy/crds/ohmyglb.absa.oss_gslbs_crd.yaml
-	kubectl apply -f deploy/crds/ohmyglb.absa.oss_v1beta1_gslb_cr.yaml
-	operator-sdk up local --namespace=test-gslb
+	kubectl apply -f ./deploy/crds/ohmyglb.absa.oss_v1beta1_gslb_cr.yaml
+	operator-sdk run --local --namespace=test-gslb --enable-delve
```

Comment:
@somaritane how was the experience with Delve enablement ?

Reply:
> @somaritane Would it be possible to provide a simple sample Grafana dashboard showing off the existing metrics with a complimentary visualisation?

@donovanmuller Good point, I'll try to publish one, maybe with sample local snapshot.

Reply:
> @somaritane how was the experience with Delve enablement ?

@ytsarev, no issues, helped me to debug some corner cases.
Actually, you need to have delve installed for that, so I'll probably make it optional, and update the docs for that.